persistence/src/main/java/de/cyface/persistence/AbstractCyfaceMeasurementTable.java

```diff
@@ -15,14 +12,17 @@
 /**
  * Abstract base class for all Cyface measurement tables implementing common functionality.
  *
- * @author Klemens Muthmann, Armin Schnabel
- * @version 1.0.0
+ * @author Klemens Muthmann
+ * @author Armin Schnabel
+ * @version 1.1.0
  * @since  1.0.0
  */
 public abstract class AbstractCyfaceMeasurementTable implements CyfaceMeasurementTable {
 
-    private static final String TAG = AbstractCyfaceMeasurementTable.class.getName();
-    private static final String maxQueryLimit = "10000"; // Else we get "Cursor Window: Window is full: requesting allocation ..." error if e.g. 157k unsynced SPs
+    /**
+     * Loading all entries at once seems slower than loading it in chunks of 10k entries (#MOV-248).
+     */
+    public static final int DATABASE_QUERY_LIMIT = 10_000;
```

Comment:
Same as above

Reply:
Same problem, the test cannot be moved (or am I missing something?).